charts/aws-ebs-csi-driver/templates/_node-windows.tpl

```diff
@@ -117,23 +148,27 @@ spec:
               exec:
                 command: ["/bin/aws-ebs-csi-driver", "pre-stop-hook"]
         - name: node-driver-registrar
-          image: {{ printf "%s%s:%s" (default "" .Values.image.containerRegistry) .Values.sidecars.nodeDriverRegistrar.image.repository .Values.sidecars.nodeDriverRegistrar.image.tag }}
-          imagePullPolicy: {{ default .Values.image.pullPolicy .Values.sidecars.nodeDriverRegistrar.image.pullPolicy }}
+          image: {{ printf "%s%s:%s" (default "" (.Values.image).containerRegistry) (default "public.ecr.aws/eks-distro/kubernetes-csi/node-driver-registrar" (((.Values.sidecars).nodeDriverRegistrar).image).repository) (default "v2.9.0-eks-1-28-6" (((.Values.sidecars).nodeDriverRegistrar).image).tag) }}
```

Comment:
I'm against putting default sidecar tags in the chart that we would have to risk changing every release.

Reply:
It seems like this default value is not relevant to the chart tester CI job failing: [Additional DaemonSets feature #1722](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/1722)

Reply:
I dislike it, but all default values (including tags) must be duplicated into the chart for this support to work. Otherwise, users upgrading with `--reuse-values` will end up with an inconsistent (or in the case of tags, broken) experience.